UKHomeOffice/sgmr-service
# NMSW-45 Update contact numbers

### AC / Description of changes

GIVEN I am viewing the help page (either from the nav, or from the voyage start button)
WHEN I scroll to the phone number section
THEN I see a list of links, one for each major region

WHEN I click on one of those links
THEN I see the phone number for that region below the link text
IF the region has multiple phone numbers
THEN I see a table that shows the location, the number(s), and the opening hours (if provided)
IF the region has a single phone number
THEN I see just the phone number

### To Test

- Run app locally
- Go to localhost:8080
- Sign in
- Go to the help page in the navigation
- > You should see details containing the new number (In document attached to ticket)
- > The South contact numbers should be in a table with location, number and operating hours (If applicable)
- Go to the dashboard (/voyage-plans)
- Click on Create a new voyage plan
- > The new number in details should appear here too

### Notes

Attached to the ticket is a PDF with different designs - Option 2 was preferred.

---

**Remember to merge to feature branches not `master`.**
